index.html

```diff
@@ -121,7 +121,7 @@ <h2 class="m-5">Let wus poppin' take care of you by providing concerts, comdedy
     </div>
 
     <!-- thumbnail -->
-  <div class=" mt-5 d-flex justify-content-center flex-wrap">
+  <div class="container mt-5 d-flex justify-content-center flex-wrap">
     <div class="card p-2 border-dark col-sm-6" style="width: 25rem;">
       <img src="IMG/pexels-pixabay-258804.jpg" class="card-img-top" alt="image-of-crowd ">
```

Comment:
Don't need to add hyphens for things like alt tags, just use normal conversational grammar. https://supercooldesign.co.uk/blog/how-to-write-good-alt-text